TaylorFramework/Modules/finder/ErrorPrinter.swift

```diff
@@ -23,10 +23,10 @@ struct ErrorPrinter {
     }
     
     func printWrongFilePath(wrongPath: FilePath) {
-        printer.printError("File path not exist or type is wrong \(wrongPath)")
+        printer.printError("File path doesn't exist or type is wrong \(wrongPath)")
```

Comment:
I think this should go like `File path '\(wrongPath)' does not exist or type is wrong`